### DISCUSSION TITLE
Documentation improvements for Table Cell Edit Feature

### DISCUSSION THREAD

**Discussion by JIBIN-P** [primefaces/community#1133] [PrimeNG] [answered]
The documentation is lacking [pEditableColumnRowIndex] or even a simple example of how to get the changes onCellEdit and missing documentation for p-cellEditor, pEditableColumnRowIndex. I would love to make these changes and provide a more intermediate example using these features of the cell-edit for the table component.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for the valuable feedback we're continuously seeking ways to improve our documentation. We'll add this example in the future. Could you please write down some use cases you're using? It would be helpful to understand and prepare examples for the common use cases.
